templates/launch.html

```diff
@@ -38,69 +49,69 @@ <h1>Launch a survey</h1>
         </span>
     </div>
 
-    <div class="field-container">
+    <div class="hidden-field-container">
         <span>
             <label for="schema-url">Schema URL</label>
             <input id="schema-url" name="schema_url" type="text" class="qa-schema_url">
         </span>
     </div>
 
-    <div class="field-container" id="schema-url-name">
+    <div class="hidden-field-container" id="schema-url-name">
         <span>
             <label for="schema-url-name-input">Schema Name</label>
             <input id="schema-url-name-input" name="schema_name" type="text" class="qa-schema_url_name" disabled>
         </span>
     </div>
```

Comment:
Can this now not be removed?. In v2 spec, if you have a URL, you don't need `schema_name`. `schema_name / eq_id/form_type` was mandatory in v1.

Reply:
Removed this